library/src/main/java/com/mux/video/upload/api/MuxUpload.kt

```diff
@@ -293,13 +294,15 @@ class MuxUpload private constructor(
      * @param videoFile a File that represents the video file you want to upload
      */
     @Suppress("unused")
-    constructor(uploadUri: String, videoFile: File) : this(Uri.parse(uploadUri), videoFile)
+    constructor(uploadUri: String, videoFile: File)
+            : this(Uri.parse(uploadUri), videoFile)
 
     private var manageTask: Boolean = true
     private var uploadInfo: UploadInfo = UploadInfo(
       // Default values
       remoteUri = uploadUri,
       file = videoFile,
+      standardizedFilePath = "",
```

Comment:
What do y'all think of using `java.util.Optional` here to indicate no path vs an empty string?

Reply:
We can just use `?`, which yeah would be a good idea